### PR TITLE
add info on priority inbox

### PR DESCRIPTION
**Pull request by khorne3**
- Minor fixes to `getting-started.md` -- I forgot the frontmatter, and I updated the title to better reflect the content
- Added info on the priority inbox feature

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team

### CODE REVIEW

**Comment by s-santillan** [comment]
> I forgot the frontmatter

Something a linter or publishing check should be able to catch. `pre-commit` bash script maybe (We'd have to provide this ourselves, there is no out-of-the-box solution.)

File: docs/semgrep-assistant/overview.md [hunk 1]
```diff
@@ -77,6 +77,15 @@ You can view findings for which Assistant has generated autofix snippets in Semg
 
 The finding's detail page also includes a link to the PR or MR comment with the autofix; you can go to the PR or MR, where you can directly commit the autofix. <!-- #10516 --> <!-- This isn't unique to Assistant; it's available for autofix and PR/MR comments in general. -->
 
+### Priority inbox
+
+Semgrep sends weekly emails with information on the top three backlog tasks according to Assistant. These emails are sent out on Monday to all organization admins.
```
Comment (s-santillan) [inline docs/semgrep-assistant/overview.md:82]: Is this for all products purchased by or enabled for the user?

**Comment by s-santillan** [comment]
LGTM, let's wait for Jack's approval :)

**Review by s-santillan — APPROVED** [review]

File: docs/semgrep-assistant/overview.md (hunk 1 above)
Comment (jsmoxon) [reply docs/semgrep-assistant/overview.md:82]: Yes Priority Inbox looks at all 3 products and will provide results if the end user has all 3.

**Review by jsmoxon — APPROVED** [review]

File: static/img/priority-inbox.png
Comment (Sly1029) [inline static/img/priority-inbox.png:1]: This screenshot is out of date. There are a couple of improvments

File: docs/semgrep-assistant/overview.md (hunk 1 above)
Comment (Sly1029) [reply docs/semgrep-assistant/overview.md:82]: Is it worth indicating that this is across products?

Comment (khorne3) [reply docs/semgrep-assistant/overview.md:82]: Added a mention that this is across all products!

**Pull request merged**